### PR TITLE
Adding min_delta setting for early stopping in training settings

### PR DESCRIPTION
**Pull request by elloza**
It is just a new parameter for the training settings to define a minimum value to be considered as an improvement in the evaluation metric.

Maybe it can be useful. It is available in frameworks like Keras and PyTorch.

Greetings and congratulations for your work.

### CODE REVIEW

**Comment by chenyuwuxin** [comment]
Thanks for your contribution! I will check it soon.

**Review by chenyuwuxin — APPROVED** [review]
